src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -1005,6 +1014,17 @@ private void enableOrDisableFields() {
 		bossBarType.setEnabled(isBoss.isSelected());
 
 		rangedAttackItem.setEnabled("Default item".equals(rangedItemType.getSelectedItem()));
+		mobBehaviourType.addActionListener(e -> {
+			celebrationSound.setEnabled(mobBehaviourType.getSelectedItem() == "Raider");
+			breedable.setEnabled(mobBehaviourType.getSelectedItem() != "Raider");
+			tameable.setEnabled(mobBehaviourType.getSelectedItem() != "Raider");
+			breedTriggerItems.setEnabled(mobBehaviourType.getSelectedItem() != "Raider");
+		});
+		LivingEntity livingEntity = new LivingEntity(modElement);

```

Comment:
There is no need to create a new `LivingEntity` there. This needs revision...

Reply:
How can I do it?

Reply:
Done